frontend/src/components/common/Label.tsx

```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
 import './Label.scss';
 interface ILabelProps {
-  children?: string;
+  content?: string;
   required?: boolean;
 }
 
-/** generic inline label element */
-export const Label = (props: ILabelProps | null | undefined) => {
+/** Generic inline label element */
+export const Label: React.FunctionComponent<ILabelProps & React.HTMLAttributes<HTMLDivElement>> = ({
```

Comment:
Labels should support all of the standard html attribute tags.

Reply:
Ooh good call